javadoc/index-files/index-5.html

```diff
@@ -62,7 +62,7 @@ <h2 class="title" id="I:G">G</h2>
 </dd>
 <dt><span class="member-name-link"><a href="../com/simulator/model/SimulationParameters.html#getAllUserNames()">getAllUserNames()</a></span> - Method in class com.simulator.model.<a href="../com/simulator/model/SimulationParameters.html" title="class in com.simulator.model">SimulationParameters</a></dt>
 <dd>
-<div class="block">Get all user names in a List<String></div>
+    <div class="block">Get all user names in a List<String></String></div>
```

Comment:
Why do we need a <String></String> at the end

Reply:
Probabably a mistake, will remove